src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -968,6 +968,19 @@ public static void EmitObject(string objectFilePath, IEnumerable<DependencyNode>
                             }
                             int size = objectWriter.EmitSymbolReference(reloc.Target, (int)delta, reloc.RelocType);
 
+                            // Emit a copy of original Thumb2 instruction that came from RyuJIT
```

Comment:
Arm only, i.e. only where these relocs exist. And, in case you want me to enclose this fragment into a target-specific if-block, I guess for Windows this is applicable too.